apps/remix-ide/src/app/files/remixd-handle.js

```diff
@@ -40,6 +40,7 @@ export class RemixdHandle extends WebsocketPlugin {
   deactivate () {
     this.fileSystemExplorer.hide()
     if (super.socket) super.deactivate()
+    this.appManager.ensureDeactivated('git')
```

Comment:
Any reason why not using `this.call('manager', 'deactivatePlugin', 'git')` ?
Use call through the engine when possible instead of direct call to a plugin.